File: content/snippets/scala2/interview_lcci/16/09_operations.scala
object Lcci1609Operations {
    final class Operations {
        def minus(a: Int, b: Int): Int = a - b
        def multiply(a: Int, b: Int): Int = a * b
        def divide(a: Int, b: Int): Int = a / b
    }

    def main(args: Array[String]): Unit = {
        val ops = new Operations
        println(ops.minus(10, 3))
        println(ops.multiply(6, -4))
        println(ops.divide(20, 5))
    }
}

File: content/snippets/typescript/interview_lcci/16/09_operations.ts
class Operations {
    minus(a: number, b: number): number {
        return a - b;
    }

    multiply(a: number, b: number): number {
        return a * b;
    }

    divide(a: number, b: number): number {
        return Math.trunc(a / b);
    }
}

const ops = new Operations();
console.log(ops.minus(10, 3));
console.log(ops.multiply(6, -4));
console.log(ops.divide(20, 5));
